dali/pipeline/operators/decoder/CMakeLists.txt

```diff
@@ -15,6 +15,9 @@
 # Get all the source files and dump test files
 if (BUILD_NVJPEG)
   list(APPEND DALI_SRCS "${CMAKE_CURRENT_SOURCE_DIR}/nvjpeg_decoder.cu")
+  list(APPEND DALI_SRCS "${CMAKE_CURRENT_SOURCE_DIR}/nvjpeg_decoder_new.cu")
```

Comment:
are those _new suffixes something to have while we test or they are meant to stay?

Reply:
It's temporary, WIP